### ISSUE TITLE
Add Functionality to the Profile Page

### ISSUE THREAD

**Issue by Prabesh-Shrestha** [Prabesh-Shrestha/embedtool#2]
The embed tool will be used in a embedded system project, when the tool is open in a code base. the Profile Section should show all the properties of the codebase: Name of the project, github link, libraries used and as much that could be shown.

**Comment by Abdullah00110** [comment]
@Prabesh-Shrestha I can do it

**Comment by Prabesh-Shrestha** [comment]
Yes, please have a look at it and try it, do send a pull request when you're done.

**Comment by Abdullah00110** [comment]
ok..
